### PR TITLE
[DOCS] Convert parameters to hugo shortcodes

### PR DESCRIPTION
**Pull request by ChrisChinchilla**
This PR converts our releases to parameters, so we can update them in one place.

Signed-off-by: ChrisChinchilla <chris@chronosphere.io>

### CODE REVIEW

**Comment by ChrisChinchilla** [comment]
@mmihic especially as you handle releases. I need to add some instructions somewhere to tell people to update version strings in docs after releases

File: site/content/cluster/create-database.sh [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-curl -X POST http://localhost:7201/api/v1/database/create -d '{
+curl -X POST {{% apiendpoint %}}database/create -d '{
```
Comment (mfesenko) [inline site/content/cluster/create-database.sh:2]: Looks like replacements inside the code block don't always work properly ([link](https://deploy-preview-3281--m3db.netlify.app/docs/cluster/kubernetes_cluster/#create-a-placement-and-namespace)):
image

File: site/content/cluster/ready-namespace.sh [hunk 2]
```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
-curl -X POST http://localhost:7201/api/v1/services/m3db/namespace/ready -d '{
+curl -X POST {{% apiendpoint %}}services/m3db/namespace/ready -d '{
```
Comment (mfesenko) [inline site/content/cluster/ready-namespace.sh:2]: Same problem with the variable not being replaced ([link](https://deploy-preview-3281--m3db.netlify.app/docs/cluster/kubernetes_cluster/#ready-a-namespace)).

File: site/content/cluster/write-metrics-1.sh [hunk 3]
```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-curl -X POST http://localhost:7201/api/v1/json/write -d '{
+curl -X POST {{% apiendpoint %}}json/write -d '{
```
Comment (mfesenko) [inline site/content/cluster/write-metrics-1.sh:2]: [Here](https://deploy-preview-3281--m3db.netlify.app/docs/cluster/kubernetes_cluster/#writing-metrics) the variable was not replaced as well.

File: site/layouts/shortcodes/operator-version.html [hunk 4]
```diff
@@ -0,0 +1 @@
+{{ .Site.Params.releases.operator }}
```
Comment (mfesenko) [inline site/layouts/shortcodes/operator-version.html:1]: nit: Maybe a newline is needed at the end of the file?

File: site/content/m3query/config/_index.md [hunk 5]
```diff
@@ -14,27 +14,27 @@ By default M3 runs two query engines:
 
 Prometheus Query Engine is the default one when calling query endpoint:
```
Comment (mfesenko) [inline site/content/m3query/config/_index.md:15]: I can't really find this page in the preview for some reason. I tried searching for "But you can switch between the two" and the search sends me [here](https://deploy-preview-3281--m3db.netlify.app/docs/m3query/config/), but the page looks different.

**Review by mfesenko — COMMENTED** [review]
It looks like all the places where `{{% apiendpoint %}}` is used in a separate `sh` script are not being replaced properly.

File: site/content/cluster/create-database.sh (hunk 1 above)
Comment (ChrisChinchilla) [reply site/content/cluster/create-database.sh:2]: Hmm, that's odd, I tested that locally 🤔

Comment (ChrisChinchilla) [reply site/content/cluster/create-database.sh:2]: And it was working before…

**Review by mfesenko — APPROVED** [review]
lgtm

**Pull request merged**